### ISSUE TITLE
Update to Docs Regarding azurerm_kubernetes_cluster, availability zones, and SLB

### ISSUE THREAD

**Issue by rudolphjacksonm** [terraform-providers/terraform-provider-azurerm#5157]
To use Availability Zones with AKS, you must use a Standard Load Balancer. You can see this behavior when trying to create a cluster using VMSS and AZs but passing Basic Load Balancer as the LB type:

```
Error: Error creating Managed Kubernetes Cluster "nhsapp-sandboxuks1" (Resource Group "nhsapp-sandboxuks1"): containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="SLBRequiredForAvailabilityZone" Message="Standard Load Balancer is required for availability zone." Target="networkProfile.loadBalancerSku"
```

The docs currently don't reflect this. I'll submit a PR to get them updated so it's clear that you have to use or migrate to the Standard SKU to enable Availability Zones with AKS.

**Comment by rudolphjacksonm** [comment]
https://github.com/terraform-providers/terraform-provider-azurerm/pull/5158

**Comment by landro** [comment]
I believe this can be closed since #5158 has been merged to master.

**Comment by tombuildsstuff** [comment]
👍 thanks for the ping @landro :)
